libs/barista-components/autocomplete/src/autocomplete-trigger.ts

```diff
@@ -319,9 +317,10 @@ export class DtAutocompleteTrigger<T>
 
     if (_optionConfig) {
       this._optionHeight = _optionConfig.height;
-      this._panelMaxHeight = Math.ceil(
-        _optionConfig.height * OPTION_TO_PANEL_HEIGHT_RATIO,
-      );
+      this._panelMaxHeight =
+        _optionConfig.height >= AUTOCOMPLETE_PANEL_MAX_HEIGHT
```

Comment:
This checks whether one single option is the same height as the panel max height - please check whether the option height is a multiple of the panel height instead. and then deduct half a option height. 
Lets say you have 3 options that fit exactly into the panelheight - i'd want to have the panel height adjusted to 2.5 option heights in that case